plugins/version.txt

```diff
@@ -1 +1 @@
-14
\ No newline at end of file
+15
```

Comment:
@animuk could you make it 16 now. In recent PR merge its got increased.

Reply:
done. Bumped up the version to 16

Reply:
@hrishin Please do the needful